### ISSUE TITLE
Can not select 'input' element after upgrade 4.11.0 

### ISSUE THREAD

**Issue by sanlengjingvv** [codeborne/selenide#719]
```HTML
<span class="ivu-checkbox">
    <span class="ivu-checkbox-inner"></span>
        <input type="checkbox" class="ivu-checkbox-input">
</span>
```

```Java
$(".ivu-checkbox").setSelected(true);
```

It can't work after upgrade selenide from 4.9.1 to 4.11.0.

> InvalidStateException Only use setSelected on checkbox/option/radio

**Comment by asolntsev** [comment]
It seem reasonable for me. You should only call "setSelected" on radiobuttons or checkboxes.

**Comment by ostap-oleksyn** [comment]
The stacktrace says it all, setSelected() works only on checkbox/option/radio elements. You can't use it on a span element. Try to use this instead:
`$("ivu-checkbox-input").setSelected(true);`

**Comment by sanlengjingvv** [comment]
`$("ivu-checkbox-input").setSelected(true);` does not work. Selenide gives this error message:
> 
Element should be visible {.ivu-checkbox-input}
Element: '<input class="ivu-checkbox-input" type="checkbox" value="on" displayed:false></input>'

In Chrome console,  I tried this:
```
document.defaultView.getComputedStyle(document.querySelector('.ivu-checkbox-input')).display
```
> "block"

**Comment by vinogradoff** [comment]
Yes, the modern JS frameworks doesn't use <input> for selecting checkbox (it is indeed invisible)- so the best solution is usually don't use setSelected but just click on visual representation of the checkbox. 

`$(".ivu-checkbox")click();  // could help`

setSelected (the same as Selenium) handles only the "old-school" plain checkboxes

- [ ] like this one
